components/raftstore/src/store/worker/split_controller.rs

```diff
@@ -252,25 +252,27 @@ impl Recorder {
     }
 }
 
-#[derive(Clone)]
-pub struct QpsStats {
+#[derive(Clone, Debug)]
+pub struct ReadStats {
+    pub flows: HashMap<u64, FlowStatistics>,
     pub region_infos: HashMap<u64, RegionInfo>,
     pub sample_num: usize,
 }
 
-impl QpsStats {
-    pub fn new() -> QpsStats {
-        QpsStats {
+impl ReadStats {
+    pub fn default() -> ReadStats {
+        ReadStats {
```

Comment:
Why not just make this a trait?

Reply:
would you like to describe more detail?

Reply:
It sounds good, do you mind submitting a pr to implement it?